doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,587 @@
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
+   |NCS| v1.1.0 does not provide support for nRF5340.
+   For nRF5340 support, check out the master branch.
+
+Highlights
+**********
+
+* Updated the :ref:`lib_fota_download` library (used by the :ref:`aws_fota_sample` sample, the :ref:`asset_tracker` application, and the :ref:`http_application_update_sample` sample) to support delta updates of the modem firmware in addition to application updates.
+* Updated the :ref:`asset_tracker` application with new features and firmware over-the-air (FOTA) support.
+* Added an :ref:`lwm2m_carrier` sample that demonstrates how to set up and use the :ref:`liblwm2m_carrier_readme` library to connect to an operator network.
+  The new :ref:`liblwm2m_carrier_readme` library provides Verizon network support.
+* Added an :ref:`lwm2m_client` that uses Zephyr's :ref:`zephyr:lwm2m_interface` library to push sensor data to a server.
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.1.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
+************************
+
+* mfw_nrf9160_1.1.0
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.50b
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v9.8.1
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.3.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.20a or later.
```

Comment:
we should remove "or later" we have tested only with 4.20a

Reply:
Removed. But it probably won't be available from the provided links anymore once we get an updated SES.